scripts/gitlab-server/README.md

```diff
@@ -192,3 +192,105 @@ Script 3 - add users to GitLab instance
 
 # Repeat this command for each new user.
 ```
+
+
+## Run the GitLab runner server deployment
+
+### Obtain the .crt file
+
+First, run the scp-from-server.sh script to obtain the .crt file needed for runner setup.
+
+```bash
+
+./scp-from-server.sh \
+    -f <server-fqdn> \
+    -d 
+
+# Parameter specifications
+    -f my-gitlab-server.eastus2.cloudapp.azure.com \  # FDQN (use DNS name label and location specified above, required)
+    -d                                                # Debug flag (optional)
+```
+
+Observe that the necessary .crt file exists within the .data folder:
+
+e.g.  
+```
+~/work/symphony/.data/ssl/server.crt
+```
+where /.data is at the root of your working folder.
+
+### Configure values for the runner setup 
+
+The runner server creation script [gitlab-runner-setup.sh](./gitlab-runner-setup.sh), is invoked by the [install.sh](./install.sh) script. 
+
+
+NOTE: As of this writing, the install.sh script contains the following:
+- (commented out) a call to gitlab-server-setup.sh
+- a set of values needed from the server VM
+- a call to gitlab-runner-setup.sh, with the necessary parameters
+ 
+ The expected parameters for gitlab-runner-setup.sh (shown below) need to be obtained from one or more locations/methods. 
+ 
+ To get values for the parameters, 
+ - Inspect the .json file exported into the /.data folder by the server setup process to observe the values in it, e.g. ./data/gitlab-server.json
+ - Capture the token obtained from running fetch-gitlab-token.sh earlier. 
```

Comment:
Remove this part, the token is now in the json file. This is also less confusing as there are 2 tokens  retrieved above (neither necessary for the setup process now with the token in the json).

Reply:
Text removed as advised, since token is no longer manually generated.